zkevm-circuits/src/copy_circuit.rs

```diff
@@ -837,6 +885,19 @@ impl<F: Field> CopyCircuitConfig<F> {
                             }
                         }
                     );
+                    #[cfg(feature = "dual_bytecode")]
+                    self.assign_copy_event(
```

Comment:
put `bytecode_map` also to `#[cfg(not(feature = "dual_bytecode"))] self.assign_copy_event` even if we don't use it.  To simplify the `cfg(feature`

Reply:
`#[cfg(not(feature = "dual_bytecode"))] self.assign_copy_event` was removed , just use old `self.assign_copy_event` with feature parameter.